web/default_factory.js

```diff
@@ -165,33 +165,33 @@ class DefaultStructTreeLayerFactory {
 class DefaultTextLayerFactory {
   /**
    * @typedef {Object} CreateTextLayerBuilderParameters
-   * @property {HTMLDivElement} textLayerDiv
    * @property {number} pageIndex
    * @property {PageViewport} viewport
    * @property {EventBus} eventBus
    * @property {TextHighlighter} highlighter
    * @property {TextAccessibilityManager} [accessibilityManager]
+   * @property {boolean} [isOffscreenCanvasSupported]
    */
 
   /**
    * @param {CreateTextLayerBuilderParameters}
    * @returns {TextLayerBuilder}
    */
   createTextLayerBuilder({
-    textLayerDiv,
     pageIndex,
     viewport,
     eventBus,
     highlighter,
     accessibilityManager = null,
+    isOffscreenCanvasSupported = false,
```

Comment:
This default value is true in the `TextLayerBuilder` constructor, so please use the same value here :-)